jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/MatchedPath.java

```diff
@@ -0,0 +1,51 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.http.pathmap;
+
+public interface MatchedPath
```

Comment:
I think it would be more intuitive to have a `getPathSpec()` method on this class, but not have it on `MatchedResource`

Reply:
That would mean MatchedResource doesn't really need to exist too.

Reply:
This change caused too many changes.
Be happy to make it in Jetty 12 though.